parser/generator.go

```diff
@@ -162,6 +162,7 @@ func (g *Generator) GenerateDockerfile() error {
 
 	for _, phase := range phases {
 		if len(phase.commands) > 0 {
+			bufferRun.WriteString("source /.bzkenv\n")
```

Comment:
I might need to use a variable but looks like it will need some lines of code for not so much in the end